models/doctype/EmailAccount/EmailAccount.js

```diff
@@ -81,13 +88,20 @@ module.exports = {
         {
             "fieldname": "smtpHost",
             "label": "SMTP Host",
-            "fieldtype": "Data",
+            "fieldtype": "Select",
+            "options": [
+                "smtp.gmail.com",
+                "smtp.mail.yahoo.com"
+            ],
             "required": 1
         },
         {
             "fieldname": "smtpPort",
             "label": "SMTP Port",
-            "fieldtype": "Int",
+            "fieldtype": "Select",
+            "options": [
```

Comment:
Why do we need options if both have same imap port & smtp port number?
Can we have this as default?